### ISSUE TITLE
Article on Loops

### ISSUE THREAD

**Issue by DarkPurple141** [DarkPurple141/teaching#5]
Anyone wanna introduce loops in a short article?

**Comment by Connor-OShea** [comment]
What's the scope of the article? Similar to your other ones?

**Comment by DarkPurple141** [comment]
Yeah more or less. Can be whatever you like. How does a loop work? Why is it useful.
